sopel/module.py

```diff
@@ -75,6 +75,13 @@
 .. versionadded:: 4.1
 """
 
+OPER = 32
+"""Privilege level for the +y channel permission
```

Comment:
Sorry, missed this one last change in the first review :disappointed: 

```suggestion
"""Privilege level for the +y/Y channel permission
```